root/src/js/lib/paywall.js

```diff
@@ -0,0 +1,23 @@
+// pageId is a unique identifier for the current project
+var paywall = function(pageId) {
+  // Set global configs to enable the pagewall and identify the current page
+  window.SEATIMESCO = window.SEATIMESCO || {};
```

Comment:
Considering that there's never going to be a SEATIMESCO in the page scope, I may change this so that it just creates the object, which I think will be a bit more succinct.

Reply:
Sure - just trying not to stomp on anything